conferences/2024/javascript.json

```
@@ -743,6 +743,21 @@
     "cocUrl": "https://nordicjs.com/code-of-conduct",
     "twitter": "@nordicjs"
   },
+  {
+    "name": "SquiggleConf",
+    "url": "https://squiggleconf.com",
```

@dimitropoulos reminded me after the form, we do have year-specific URLs, whoops:

```suggestion
    "url": "https://2024.squiggleconf.com",
```

---

That's Ok. I'll merge it with the general URL. The website is still year specific.